src/server/routes/api/thumbnail.js

```diff
@@ -6,6 +6,13 @@ import fetch from 'node-fetch';
 import { getNetworkDocURL, getSnapshotDocURL, removeQuotes } from './history';
 import cytosnap from 'cytosnap';
 
+const snap = cytosnap({
+  puppeteer: {
+    args: ['--headless', '--disable-dev-shm-usage', '--no-sandbox', '--disable-setuid-sandbox', '--no-zygote']
+  }
+});
+
+const snapStart = snap.start();
```

Comment:
So the idea here is to keep one instance of cytosnap and use the same instance everytime `runCytosnap` is called?

Reply:
Yes.  Each cytosnap instance has an instance of chrome, which is heavy